src/components/SelectField/SelectField.js

```diff
@@ -0,0 +1,80 @@
+import React, {PropTypes} from 'react'
+import {colors} from '../../variables'
+
+const defaultStyle = {
+  backgroundColor: colors.white,
+  borderRadius: '0.4rem',
+  border: '1px solid',
+  borderColor: colors.silver,
+  height: '3.3rem',
+  width: '100%'
+}
+const defaultLabelStyle = {
+  display: 'block',
+  fontWeight: 600,
+  marginBottom: '0.5rem',
+  maxWidth: '100%'
+}
+
+/**
+ * Can be given options to list in the select dropdown (assigned as optionValues)
+ */
+const SelectField = (props) => {
+  let {optionValues, options} = props
+  if (!optionValues) {optionValues = options}
+
+  let borderColorStyle = Object.assign({},
+    props.warning ? {borderColor: colors.goldenTainoi} : {},
+    props.error ? {borderColor: colors.amaranth} : {}
+  )
+  let textColorStyle = Object.assign({},
+    props.warning ? {color: colors.goldenTainoi} : {},
+    props.error ? {color: colors.amaranth} : {}
+  )
+
```

Comment:
one is color and the other is borderColor - because of the way object assign works, we need to use separate variables for these because we want a style object that is specific